content/ja/docs/concepts/containers/runtime-class.md

```diff
@@ -40,7 +37,7 @@ RuntimeClassは、クラスター全体で同じ種類のノード設定であ
 設定が異なるノードをサポートするには、[スケジューリング](#scheduling)を参照してください。
 {{< /note >}}
 
-RuntimeClassの設定は、RuntimeClassによって参照される`ハンドラー`名を持ちます。そのハンドラーは正式なDNS-1123に準拠する形式のラベルでなくてはなりません(英数字 + `-`の文字で構成されます)。
+RuntimeClassの設定は、RuntimeClassによって参照される`ハンドラー`名を持ちます。そのハンドラーは有効な[DNSラベル名](/docs/concepts/overview/working-with-objects/names/#dns-label-names)でなくてはなりません。
```

Comment:
```suggestion
RuntimeClassの設定は、RuntimeClassによって参照される`ハンドラー`名を持ちます。そのハンドラーは有効な[DNSラベル名](/ja/docs/concepts/overview/working-with-objects/names/#dns-label-names)でなくてはなりません。
```

Reply:
I updated the link address with Japanese page.
I have checked to jump on Netlify preview.